### ISSUE TITLE
Feedback does not reflect in-game results

### ISSUE THREAD

**Issue by nfreader** [tgstation/tgstation#33800]
[Round ID]: #80680 (but seriously, all of them)

Compare these results from the round-end screen:
```Shift Duration: 48 minutes and 42 seconds
    Station Integrity: 98.3%
    Total Population: 112

    Evacuation Rate: 53 (47.3%)
    (on emergency shuttle): 38 (33.9%)
    Survival Rate: 90 (80.4%)
```

To the Statbus page for this round: 
https://atlantaned.space/statbus/round.php?round=80680

Specifically: 
* duration is off by about 15 minutes,
* The survival/escape population is not correct,
* Station integrity isn't in feedback

**Comment by FantasticFwoosh** [comment]
They very rarely listen anyway. 🤡

**Comment by AnturK** [comment]
They use slightly different count methods, need to be unified.

**Comment by Jordie0608** [comment]
This was fixed in #34465
